polkadot/node/core/pvf/tests/it/main.rs

```diff
@@ -153,6 +152,68 @@ async fn terminates_on_timeout() {
 	assert!(duration < TEST_EXECUTION_TIMEOUT * JOB_TIMEOUT_WALL_CLOCK_FACTOR);
 }
 
+// What happens when the prepare worker dies in the middle of a job?
+//
+// To avoid interfering with other running tests or processes, this test is ignored and must be run
```

Comment:
how does it interfere? with the `killall` I would assume.
Could we spawn a process in a new pid namespace to avoid this?

Reply:
Rejigged this, see https://github.com/paritytech/polkadot-sdk/pull/2009/commits/54178182b900c66fc7b7da2f5f80f09ea810383f. Slightly hacky, but works, and preferable to having ignored tests in the repo.